includes/Core/Storage/Post_Meta_Setting.php

```diff
@@ -106,6 +106,19 @@ protected function get_sanitize_callback() {
 		return null;
 	}
 
+	/**
+	 * Checks whether a post meta exists or not.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param int    $post_id  Post ID.
+	 * @param string $meta_key Meta key.
+	 * @return bool True if the meta key exists, otherwise false.
+	 */
+	public function has( $post_id, $meta_key ) {
+		return metadata_exists( 'post', $post_id, $meta_key );
```

Comment:
Let's update this to use the meta key via the class constant here like we do in the rest of the class – only the `$post_id` is needed here.

Reply:
:+1: updated.